test/e2e/core.go

```diff
@@ -97,3 +123,29 @@ func logCore(cmds []command, hosts []string, dir, provider string) {
 	}
 	wg.Wait()
 }
+
+func findKubeletSystemdService(provider string, hosts ...string) (bool, error) {
```

Comment:
@ixdy What do you mean by "never use the error return value"? If there's an error, a message will be printed.